### ISSUE TITLE
Issue in deleting related tables

### ISSUE THREAD

**Issue by vijayrathore8492** [nocodb/nocodb#1861]
**Please enter the following details**

If you delete childTable, parent table will still show the relationship column. And it will not load anymore.
There are issue on metadata page as well after this.

```
NocoDB used as docker : false
NocoDB version : 0.90.7
Database used in NC_DB URL  : sqlite3
Project was created  by clicking : New Project by connecting to external database
Database on which spreadsheet is created : pg
OS on which NocoDB is running : MacOS
Node.js version if running as node : 12.22.10
Database version : 
```

**Steps To Reproduce**

1. Create a table `table1`
2. Create another table `table2`
3. Create relationship between tables as `table1` has many `table2`
4. Delete `table2`
5. Go to `table1`, it will not load any data anymore and relationship column would still exist.
6. You would get error on project metadata tab as well.

**Expected Behavior**

`table1` should load and relationship column should not be there anymore.

**Screenshots**

Join our discord : https://discord.gg/5RgZmkW for realtime help.

**Comment by o1lab** [comment]
@vijayrathore8492 : please attach any logs from the docker or browser console.

I test and it works with a caveat in mysql though.

**Comment by wingkwong** [comment]
Fixed in `0.90.8`.
